tools/manage_clvm.py

```diff
@@ -330,20 +387,36 @@ def build() -> int:
 
                 generated_bytes = ClvmBytes.from_hex_bytes(hex_bytes=generated_paths.hex.read_bytes())
                 reference_paths.hex.write_bytes(generated_bytes.hex)
+
+                # Only add hashes to json file if they didn't already exist in it
+                hash_stems.append(reference_paths.hash)
+                if reference_paths.hash not in new_hashes:
+                    new_hashes[reference_paths.hash] = ClvmBytes.from_clvm_paths(
```

Comment:
Raising in a separate discussion here since I almost forgot about this.

https://github.com/Chia-Network/chia-blockchain/pull/14996#discussion_r1163087666
> It's not full paths. but rather names of files. This restricts us to only one clvm program per name in the project, but that's probably a good restriction.

This should be enforced directly, not accidentally. As is, two copies of the same file could exist and validate against this single hash, I would guess? Unlikely, sure, but it's nice to avoid corners and uncertainty when glancing at a system.

Reply:
I'm going to leave this open for another reviewer to read.  If they do and are not concerned and provide an approval, they are welcome to close this.